tox.ini

```
@@ -10,16 +10,16 @@ setenv =
 passenv = OPENEDX_REPO OPENEDX_REFERENCE_REPO OPENEDX_RELEASE
 
 deps =
-    django18: Django>=1.8,<1.9
-    django111: Django>=1.11,<2.0
+    django22: Django==2.2.12
```

nit: please update to match the [juniper django version](https://github.com/edx/edx-platform/blob/open-release/juniper.1/requirements/edx/django.txt#L1):

```suggestion
    django22: Django==2.2.13
```